src/main/java/ioanandroflorea/driver/BrowserManager.java

```diff
@@ -0,0 +1,14 @@
+package ioanandroflorea.driver;
```

Comment:
Instead of adding another package, can you please rename the old one ioanaandroflorea to ioanandroflorea? You can choose which one you like the most. The idea is to have only one package with name for each student. Thanks!

Reply:
I think I resolved it.

Reply:
Here it will be needed a merge with main branch most probably before for you to be able to see the other package that I was referring to.